app/models/user.rb

```diff
@@ -14,8 +14,12 @@ class User < ApplicationRecord
   mount_uploader :avatar, AvatarUploader
 
   belongs_to :osbb, optional: true
+
   has_one :address, as: :addressable, dependent: :destroy
 
+  has_many :billing_contracts, dependent: :nullify
```

Comment:
`dependent: :destroy` this are specific per user associations we dont need them without user

Reply:
No. User assigns needed contract